### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by Obehi234**
Database set up completed.

### CODE REVIEW

File: app/src/main/java/com/example/streammoviesapplication/data/db/MoviesDatabase.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.example.streammoviesapplication.data.db
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import com.example.streammoviesapplication.data.trendingMovies.localData.TrendingMoviesEntity
+
+@Database(entities = [TrendingMoviesEntity::class], version = 1, exportSchema = false)
+abstract class MoviesDatabase : RoomDatabase() {
+    abstract fun movieDao() : TrendingMoviesDao
```
Comment (Ruthy4) [inline app/src/main/java/com/example/streammoviesapplication/data/db/MoviesDatabase.kt:9]: same here

Comment (Ruthy4) [reply app/src/main/java/com/example/streammoviesapplication/data/db/MoviesDatabase.kt:9]: you will have other daos

File: app/src/main/java/com/example/streammoviesapplication/data/db/TrendingMoviesDao.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.example.streammoviesapplication.data.trendingMovies.localData.TrendingMoviesEntity
+import com.example.streammoviesapplication.utils.Constants.MOVIE_DATABASE
+
+@Dao
+interface TrendingMoviesDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertTrendingMoviesToDatabase(movies: List<TrendingMoviesEntity>)
```
Comment (Ruthy4) [inline app/src/main/java/com/example/streammoviesapplication/data/db/TrendingMoviesDao.kt:14]: rename this to `insertTrendingMovies`. we already know dao is attached to database

**Review by Ruthy4 — APPROVED** [review]

File: app/src/main/java/com/example/streammoviesapplication/data/db/TrendingMoviesDao.kt (hunk 2 above)
Comment (Obehi234) [reply app/src/main/java/com/example/streammoviesapplication/data/db/TrendingMoviesDao.kt]: done. I thought we were going to have only one dao thats why i used the previous naming conventions.

**Pull request merged**